### PR TITLE
firebase actions

### PR DESCRIPTION
**Pull request by ManthanKeim**
test @harshpavuluri

### CODE REVIEW

File: .github/workflows/firebase-hosting-merge.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      - master
+jobs:
+  build_and_deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: npm ci && npm run build
+      - uses: FirebaseExtended/action-hosting-deploy@v0
+        with:
+          repoToken: '${{ secrets.GITHUB_TOKEN }}'
```
Comment (harshpavuluri) [inline .github/workflows/firebase-hosting-merge.yml:17]: Did you add a GITHUB Token to a the repository? You can make one pretty easily. Make sure you watch the expiration time

File: .github/workflows/firebase-hosting-merge.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+          firebaseServiceAccount: '${{ secrets.FIREBASE_SERVICE_ACCOUNT_IRONHACKS_C406A }}'
```
Comment (harshpavuluri) [inline .github/workflows/firebase-hosting-merge.yml:18]: Same thing here but for the Firebase service account. This gives the authentication protocol to allow the CD to deploy onto firebase

**Review by harshpavuluri — CHANGES_REQUESTED** [review]
Check the Github secrets here....If we don't have access to the repo settings, I don't these have been included

**Comment by ManthanKeim** [comment]
You mean access to master branch ?

**Comment by harshpavuluri** [comment]
> You mean access to master branch ?

No, I mean we need to be able to create Github Secrets to make sure that we can encrypt the Firebase Service Account. See below for info on secrets.

https://docs.github.com/en/actions/security-guides/encrypted-secrets

We need to be able to at least be able to add the Firebase Service account to this so we can maintain security and be able to deploy automatically onto Firebase

**Comment by harshpavuluri** [comment]
`npm ci` should be ok. That command just sets up the automated environment (Github actions for us) with the packages (very similar to npm install but much clear to streamline things). From what I noticed `npm run build:dev` was working for me on the CI but you need a deployment

**Pull request merged**